easybuild/easyconfigs/s/span-lite/span-lite-0.10.3-GCC-11.2.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'span-lite'
+version = '0.10.3'
+
+homepage = 'https://github.com/martinmoene/span-lite'
+description = """span lite is a single-file header-only library to provide a bounds-safe view for sequences
+ of objects. The library provides a C++20-like span for use with C++98 and later. """
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
```

as a header-only library it should probably live at `GCCcore` level
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
```

---

I think GCC is correct, GCC is more minimal than GCCcore. 
GCCcore pulls in binutils.  Are binutils needed for header files?

---

You have that backwards: GCC = GCCcore + binutils + zlib. It sits at the same level as `intel-compilers`, whilst `GCCcore` is a base for both